app/views/users/_metadata.html.erb

```diff
@@ -1,34 +1,78 @@
-<% if @user.employment_title.present? || (@user.looking_for_work_publicly == true && @user.looking_for_work == true) || @user.education.present? %>
-  <div class="profile-header__bottom fs-base">
-    <% if @user.education.present? %>
-      <div class="crayons-definition">
-        <strong class="crayons-definition__title">Education</strong>
-        <p class="crayons-definition__value"><%= @user.education %></p>
-      </div>
-    <% end %>
+<% if FeatureFlag.enabled?(:profile_admin) %>
+  <% if (header_fields = @profile.ui_attributes_for(area: :header)).present? %>
+    <div class="profile-header__bottom fs-base">
+      <% header_fields.each do |title, value| %>
+        <div class="crayons-definition">
+          <strong class="crayons-definition__title">
+            <%= sanitized_sidebar title.titleize %>
+          </strong>
+          <p class="crayons-definition__value">
+            <%= sanitized_sidebar value %>
+          </p>
+        </div>
+      <% end %>
+      <%# As we migrate to more generalized profiles, we don't want to break DEV %>
+      <% if SiteConfig.dev_to? %>
```

Comment:
In the case that we are on DEV, we want to preserve the employer and looking for work fields; they need to be hardcoded at the moment.